### PR TITLE
KOGITO-9512 - Unify logic-swf-* image description with labels and fix…

### PR DESCRIPTION
**Pull request by MarianMacik**
… BDD tests

**JIRA:** [KOGITO-9512](https://issues.redhat.com/browse/KOGITO-9512)

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

- <b>(Re)run Jenkins tests</b>  
  Please add comment: <b>Jenkins [test|retest] this</b>

- <b>Prod tests</b>  
  Please add comment: <b>Jenkins (re)run [prod|Prod|PROD]</b>
</details>

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

File: tests/features/openshift-serverless-logic/logic-swf-devmode.feature [hunk 1]
```diff
@@ -6,7 +6,6 @@ Feature: Serverless Workflow devmode images
     Scenario: verify if all labels are correctly set on logic-swf-devmode-rhel8 image
       Given image is built
         Then the image should contain label io.openshift.expose-services with value 8080:http,5005:http
-        AND the image should contain label maintainer with value kogito <bsig-cloud@redhat.com>
```
Comment (MarianMacik) [inline tests/features/openshift-serverless-logic/logic-swf-devmode.feature:9]: We can discuss this, apparently in prod we set this to `Red Hat, Inc.`, so we might check it if needed, but I am not sure it is worth it.

Comment (spolti) [reply tests/features/openshift-serverless-logic/logic-swf-devmode.feature:9]: yeah. probably better to add Red Hat instead Kogito for prod, but the email we can keep the same.

Comment (MarianMacik) [reply tests/features/openshift-serverless-logic/logic-swf-devmode.feature:9]: @spolti Considering that we are not checking this on other images at all, do I have to leave this step or can be removed, wdyt? We are currently not setting any maintainer, is it needed?

Comment (rgdoliveira) [reply tests/features/openshift-serverless-logic/logic-swf-devmode.feature:9]: @MarianMacik @spolti about the maintainer label, let me paste here what I mentioned in the channel.

We have not set yet any maintainer label yet for prod images (neither for RHPAM nor Kogito images).

- kogito-runtime-jvm-overrides.yaml (community) has the maintainer label https://github.com/kiegroup/kogito-images/blob/1.13.x/kogito-runtime-jvm-overrides.yaml
- rhpam-kogito-runtime-jvm-rhel8-overrides.yaml (prod), doesn't have the maintainer label https://github.com/kiegroup/kogito-images/blob/1.13.x/rhpam-kogito-runtime-jvm-rhel8-overrides.yaml. 

If we want to start setting maintainer label for the OSL prod images, I guess we should decide what to use. Just as an example, OpenShift Serverless team is using `maintainer="serverless-support@redhat.com"`

**Review by davidesalerno — APPROVED** [review]

File: tests/features/openshift-serverless-logic/logic-swf-devmode.feature (hunk 1 above)
Comment (ricardozanini) [reply tests/features/openshift-serverless-logic/logic-swf-devmode.feature:9]: We can keep bsig-cloud@redhat for now. Can we change this in later releases?

**Review by ricardozanini — APPROVED** [review]

**Review by rgdoliveira — APPROVED** [review]

**Comment by kie-ci1** [comment]
**PR job** `#988` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/988/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
<br/>        -------------------------------------------------------------<br/>        [ERROR] Can't merge source into Target. Please rebase PR branch.<br/>        -------------------------------------------------------------<br/>        Source: git://github.com/MarianMacik/kogito-images KOGITO-9512<br/>        Target: d3d111e0 [main] Update version to 2.0.0-snapshot<br/>        -------------------------------------------------------------<br/>        <br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Validate CeKit Image and Modules descriptors)<br/>Stage "Validate CeKit Image and Modules descriptors" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Build & Test Images)<br/>Stage "Build & Test Images" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[INFO] Clean workspace<br/>[Pipeline] cleanWs<br/>[WS-CLEANUP] Deleting project workspace...<br/>[WS-CLEANUP] Deferred wipeout is used...<br/>[WS-CLEANUP] done<br/>[Pipeline] echo<br/>[INFO] Workspace cleaned<br/>[Pipeline] echo<br/>[INFO] Cleanup Maven artifacts<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.m2/repository<br/>[Pipeline] echo<br/>[INFO] .m2/repository cleaned<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/988/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Review by spolti — APPROVED** [review]

**Pull request merged**